src/config.ts

```diff
@@ -51,6 +51,18 @@ export const WECHATY_PUPPET_SERVICE_ENDPOINT = () => {
   return undefined
 }
 
+export const GET_CHATIE_ENDPOINT_LIST = () => {
+  const serviceDiscoveryEndpointList = process.env.SERVICE_DISCOVERY_ENDPOINT_LIST // e.g. https://aaa.aaa,http://bbb.bbb
+  if (serviceDiscoveryEndpointList) {
+    return serviceDiscoveryEndpointList.split(',')
+  }
+  return [
+    'https://api.chatie.io',
+    'https://chatieio.herokuapp.com',
+    'http://68.79.16.140',  // from @windmemory,
```

Comment:
Could we use a DNS name for this instead of the IP?

For example, chatie-api-proxy.api.juzi.bot with `https` enabled?